src/app/modules/customer-managment/components/managment-customer-projects/managment-customer-projects.component.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../../../../styles/colors.scss';
```

Comment:
file name has a typo. It needs to be 
management-customer-projects, yours is **managment**-customer-projects